srv/modules/runners/deepsea.py

```diff
@@ -17,12 +17,12 @@ def version(**kwargs):
     format_ = kwargs['format'] if 'format' in kwargs else 'plain'
 
     if format_ == 'json':
-        ver = re.findall(r'^\d\.\d\.?\d?', DEEPSEA_VERSION)
+        ver = re.findall(r'(^\d+(\.\d+){1,2})', DEEPSEA_VERSION)
         offset = re.findall(r'\+\d+', DEEPSEA_VERSION)
         hash_ = re.findall(r'[\w]{7,8}$', DEEPSEA_VERSION)
 
         return {'full_version': DEEPSEA_VERSION,
-                'version': ver[0] if ver else '0.0.0',
+                'version': ver[0][0] if ver[0] else '0.0.0',
```

Comment:
```suggestion
                'version': ver[0][0] if ver and ver[0] else '0.0.0',
```

(ver will be an empty list if the version doesn't match the regex at all)